EndlessLauncher/App.xaml.cs

```diff
@@ -47,11 +48,39 @@ protected override void OnStartup(StartupEventArgs e)
                 Debug.SetDebugSimulatedError(errorCode);
             }
 
-            mutex = new Mutex(true, "{5E80890D-A4E6-4B8C-B123-FFD89450547F}", out bool isNew);
-            if (!isNew)
+            if (e.Args.Length > 0)
```

Comment:
We are dealing with this arguments separately from the logging and debugging arguments above (which also are Unix's hyphenized flags, compared to these), should we move everything to a loop to deal with all the flags? Or use a [command line library](https://www.nuget.org/packages/System.CommandLine), probably too much for this.

Reply:
Oh, that's hilarious, my eyes completely skipped over the existing debugging / logging arguments. Yeah, I'll fix that, and I should change the format to match :)